src/unix/notbsd/linux/other/mod.rs

```diff
@@ -689,7 +689,7 @@ pub const NFPROTO_NETDEV: ::c_int = 5;
 
 // linux/netfilter/nf_tables.h
 cfg_if!{
-    if #[cfg(any(target_arch = "arm", target_arch = "aarch64"))] {
+    if #[cfg(target_arch = "aarch64")] {
```

Comment:
All architectures use 256 now: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/netfilter/nf_tables.h#L6

Reply:
`Sparc64` runs older kernel on Debian (because of https://github.com/rust-lang/libc/commit/ac4afe746f23766204fe2545c19e18d4ffb2d4d5) https://github.com/rust-lang/libc/blob/39ca01c8b3631db48c8e390f93f62e52b51a68b9/ci/docker/sparc64-unknown-linux-gnu/Dockerfile#L1

It should be upgraded if it works.